libc/src/__support/CPP/type_traits/remove_all_extents.h

```diff
@@ -16,7 +16,7 @@
 namespace LIBC_NAMESPACE::cpp {
 
 // remove_all_extents
-#if LIBC_HAS_BUILTIN(__remove_all_extents)
+#if __has_builtin(__remove_all_extents)
```

Comment:
We no longer need to include src/__support/macros/config.h here thanks to this change.

Reply:
Done.